tests/utils.go

```diff
@@ -49,23 +51,43 @@ func init() {
 }
 
 func createArtifactorySearchManager() {
-	testsSearchService = services.NewSearchService(httpclient.NewDefaultHttpClient())
+	client, err := httpclient.ClientBuilder().Build()
+	if err != nil {
+		log.Error(fmt.Sprintf(HttpClientCreationFailureMessage, err.Error()))
+		os.Exit(1)
+	}
+	testsSearchService = services.NewSearchService(client)
 	testsSearchService.ArtDetails = getArtDetails()
 }
 
 func createArtifactoryDeleteManager() {
-	testsDeleteService = services.NewDeleteService(httpclient.NewDefaultHttpClient())
+	client, err := httpclient.ClientBuilder().Build()
+	if err != nil {
+		log.Error(fmt.Sprintf(HttpClientCreationFailureMessage, err.Error()))
+		os.Exit(1)
+	}
```

Comment:
The above lines are duplicated three times in this file.
I suggest we put them in a private function and reuse it.